config.yml

```diff
@@ -1,16 +1,20 @@
-load:
-  type: constant
-  stages:
-  - rate: 1
-    duration: 30
-api: chat
+
 server:
-  type: vllm
-  model_name: HuggingFaceTB/SmolLM2-135M-Instruct
-  base_url: http://0.0.0.0:8000
-  ignore_eos: true
-tokenizer:
-  pretrained_model_name_or_path: HuggingFaceTB/SmolLM2-135M-Instruct
+  vllm:
+    base_url: http://0.0.0.0:8000
+    ignore_eos: false
+    load:
```

Comment:
Lets discuss this. Not sure if I understand the reason for combining load config and server configs.

Reply:
Reverted